tests/testthat/test-lists.R

```diff
@@ -22,3 +22,39 @@ test_that("chmp_lists curl options work", {
 
   expect_error(chmp_lists(x, timeout_ms = 1), "Timeout was reached")
 })
+
+x5 <- ChmpClient$new(dc = "us5") # Current Location
+
+test_that("chmp_post_lists with vcr", {
+  skip_on_cran()
+  skip_on_travis()
+
+  vcr::use_cassette("chmp_post_lists", {
+
+    rand_string <- function() {
+      paste(sample(c(letters, 1:9), 9), collapse = "")
+    }
+
+    aa <- chmp_post_list(x5,
+                         list_id = Sys.getenv("MAILCHIMP_LISTID"),
+                         email_address = paste0(rand_string(), "@", rand_string(), ".com"),
+                         status = "subscribed")
+
+    expect_is(aa, "list")
```

Comment:
Only feedback on this vcr stuff is to put your expectations outside of the `use_cassette` block. So do:

```r
vcr::use_cassette("xxx", {
   aa <- chmp_post_list(some code)
})

expect_is(aa, "list")
# etc...
```

putting expectionas outside the vcr block makes it so that when an error occurs when running tests, testthat points more accurately to the correct line of code

Reply:
Oh wow, thanks for the feedback! I should've checked the vcr documentation more thoroughly, sorry about that😅. I think my latest commit addresses those changes.

Also, I'd be happy to take over as chimpr maintainer but, as of now, I'm not planning on adding too many new features (unless requested). Is that okay?